app/services/eyeshade/referral_totals.rb

```diff
@@ -0,0 +1,27 @@
+class Eyeshade::ReferralTotals < Eyeshade::BaseApiClient
+  def perform
+    if Rails.application.secrets[:api_eyeshade_offline]
```

Comment:
minor: could you do `return perform_offline if Rails.application.secrets[:api_eyeshade_offline]` for consistency?